### PR TITLE
Build clean app

### PR DESCRIPTION
**Pull request by meghanhade**
Rebuilds the app to help some issues that were occurring with testing and naming/app structure in branch `meghan-create-app`. 

The route structure has been changed a bit: instead of a `submit` route with multiple subroutes, it seems better now to have one user flow, but allow for an "edit mode" when a maintainer is updating or adding to an existing dataset.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (dianashk) [inline .travis.yml]: not sure we need both travis and circle. they should be capable of doing the same things.

File: config/targets.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* eslint-env node */
+module.exports = {
+  browsers: [
+    'ie 9',
+    'last 1 Chrome versions',
```
Comment (hanbyul-here) [inline config/targets.js]: Can I ask what it is doing? Are we actually testing against some browsers other than Phantom?

File: .travis.yml (hunk 1 above)
Comment (meghanhade) [inline .travis.yml]: hmm, I think you're right @dianashk. That file was autogenerated, but it looks ok to remove.

File: config/targets.js (hunk 2 above)
Comment (meghanhade) [inline config/targets.js]: This is where you can set browser compatibility preferences, to minimize the number of Javascript to ES5 transformations.

More info here: https://guides.emberjs.com/v2.14.0/configuring-ember/build-targets/

**Pull request merged**